test/testdrive/session.td

```diff
@@ -29,6 +29,14 @@ standard_conforming_strings on                     "Causes '...' strings to trea
 statement_timeout           "10 s"                 "Sets the maximum allowed duration of INSERT...SELECT, UPDATE, and DELETE operations."
 TimeZone                    UTC                    "Sets the time zone for displaying and interpreting time stamps (PostgreSQL)."
 transaction_isolation       "strict serializable"  "Sets the current transaction's isolation level (PostgreSQL)."
+max_clusters                10                     "The maximum number of clusters in the region (Materialize)."
```

Comment:
Bikeshed: should these be prepended with system: `system_max_clusters`?

Reply:
The names were chosen to be consistent with Postgres. This is from the doc with all the variables (https://www.notion.so/materialize/Default-system-limits-f36a447644d9461f8665180fc5d7c505):
> Limits will be exposed to end users via [system configuration parameters](https://www.notion.so/materialize/Server-configuration-parameters-c9aa8050a2884f67a30b4a34aae2d1bd?d=653c75a293b84890b39d5829366c3e6e#be931d29323a4c46ac3c10cdade785f2). They are named consistently with PostgreSQL parameters that describe limits, like max_connections and max_files_per_process.